### DISCUSSION TITLE
Legrand devices disconnect after few second

### DISCUSSION THREAD

**Discussion by itobald** [Koenkk/zigbee2mqtt#22734] [Devices]
Hi,

I have an issue with my Legrand device 067775/741811.
They seem to pair well but left the network after few seconds.

![Capture d’écran 2024-05-22 à 12 22 42](https://github.com/Koenkk/zigbee2mqtt/assets/73329621/6cdcda7a-2db8-4988-a55e-571dee52f52c)

Do you have any idea of the issue ?

Zibee2Mqtt 1.37.1 with Home Assistant.

**Comment by iceland12** [comment]
I'm experiencing the same issue with Legrand 412171 with Zibee2Mqtt 1.38.1 with OH4. Some of same device are correctly joined but some other not. My idea is probably their fw version are different but obviously I cannot upgrade the ones not joined.
Any idea is appreciated.

**Comment by Matriciel** [comment]
I have same problem.
I have downgraded to version 1.34.0, pairing the devices and upgraded to 1.40.1.
I have tested with most Legrand devices, all to the latest firmware and it is same problem.

**Reply by iceland12** [reply]
So you've upgraded Legrand devices to the latest firmware without using Z2M ? If yes how ?
As reported, I've some of them joined to Z2M, so I used Z2M to upgrade w/o issues. 
For other Z2M join is not working and so cannot try to do the firmware upgrade.

**Reply by Matriciel** [reply]
Firmware upgrade are does with Z2M or Legrand getaway.
If i downgrade Z2M, pairing is possible and after upgrade to latest version, device is working (and upgraded if necessary)

**Reply by Matriciel** [reply]
I have tested two devices 412170:

- One with firmware 005a (latest)
- One with firmware 0051

The first one have problem with pairing.
The workaround that i have found is:

- Launch pairing on Z2M
- Launch pairing on device. The led blink green
- When device is discovered on Z2M and led is always blinking, power off the device
- Stop Z2M pairing
- Power on the device, the led red turn off after 3s, the device is working.

The second device with older firmware is pairing as normally, without issue.

I have tested several time.

Maybe a problem with latest firmware on Legrand devices in pairing process ?
